haddockbundle.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+
```

Comment:
Can you add a brief "Usage" section? Even if it's running it without any arguments.